.github/workflows/main-pr.yaml

```diff
@@ -78,7 +80,7 @@ jobs:
         # always is fine
 
       - name: Run chart-testing (install)
-        run: ct install --all
+        run: ct install --charts charts/kong
```

Comment:
What is the reason for specifying only the `kong` chart here 🤔 ?

I can see `.Values.deployment.test.enabled` dependent manifests in `ingress` chart as well but OTOH I can see those used in the `test-run.sh` so this has to be used in a different way. It's been a while I looked at `ct` testing.

Reply:
The `.Values.deployment.test.enabled` check is for deploying resources used by `helm test` rather than `ct install`.

The `ct install` tests run the chart for each values file in `charts/<name>/ci`. Reading through I'm not sure how they work, or if I'll be able to add one for `ingress` without changing the underlying `kong` chart as `adminApiService.name` currently needs to be known in advance.

Given we have `helm test` tests for the new `ingress` chart I'd like to merge this and we can revisit `ct install` when @rainest returns